engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFBO.java

```diff
@@ -136,36 +127,29 @@ private int fetchTextureId() {
                 case LightAccumulationTexture:
                     return inputFbo.lightBufferTextureId;
             }
-
         } else {
-            logger.warning("FBOs manager has no record of an FBO named " + this.fboURN.toString());
+            logger.warning("FBOs manager has no record of an FBO named " + this.fboUrn.toString());
         }
 
         return 0;
     }
 
     /**
-     * Normally called by the FBO manager provided on construction, when the FBOs are regenerated.
+     * Normally called by the FBO manager provided, when the FBOs are regenerated.
      *
      * This method refreshes the task's reference to the FBO attachment, so that they are always up to date.
      */
     @Override
     public void update() {
-
-        inputFbo = fbosManager.get(fboURN);
-
-        // If a node taking advantage of this state change is disabled when a game is started, task is null.
-        // This is due to the task list generator not having yet called the generateTask() method.
-        if (task != null) {
-            task.setTextureId(fetchTextureId());
-        }
+        inputFbo = fboManager.get(fboUrn);
+        task.setTextureId(fetchTextureId());
     }
 
     @Override
     public String toString() {
         if (this != defaultInstance) {
             return String.format("%30s: slot %s, fbo %s, textureType %s, fboManager %s, material %s, parameter '%s'", this.getClass().getSimpleName(),
-                    textureSlot, fboURN.toString(), textureType.name(), fbosManager.toString(), materialURN.toString(), parameterName);
+                    textureSlot, fboUrn.toString(), textureType.name(), fboManager.toString(), materialURN.toString(), parameterName);
```

Comment:
I hope you don't mind too much? I like clear code, and clear and informative names.

In this case, "Singular noun + Manager" should be the pattern (It's never sheepsManager, it's always sheepManager),and so I did it. I plan on renaming the FBOsManagers to FBOManagers in a separate PR too.